src/promptflow/promptflow/batch/_batch_engine.py

```diff
@@ -236,19 +313,27 @@ async def _exec(
         self,
         line_results: List[LineResult],
         aggr_result: AggregationResult,
-        batch_inputs: List[Dict[str, Any]],
+        full_inputs: List[Dict[str, Any]],
         run_id: str = None,
         output_dir: Path = None,
         raise_on_line_failure: bool = False,
+        lines_to_run: List[int] = None,
```

Comment:
Please remove the input parameter of `lines_to_run`.
The information can be calculated from `line_results` and `full_inputs`